src/assets/data/games.json

```diff
@@ -0,0 +1,5886 @@
+{
+  "schemaVersion": "0.0.11",
+  "games": {
+    "20-minutes-till-dawn": {
+      "uuid": "57ae7cc1-b21a-4592-8415-73512a91b180",
+      "label": "20-minutes-till-dawn",
+      "meta": {
+        "displayName": "20 Minutes Till Dawn",
+        "iconUrl": "20MinutesTillDawn.jpg"
+      },
+      "distributions": [
+        {
+          "platform": "steam",
+          "identifier": "1966900"
+        }
+      ],
+      "r2modman": {
```

Comment:
Should this be something like `mod-manager` instead of `r2modman`?

Reply:
No since it's explicitly tailored for r2modman's needs currently, and I wanted the name to indicate that. (In fact, everything within the r2modman object is exported from the r2modman code).